onetl/connection/db_connection/kafka/connection.py

```diff
@@ -159,7 +159,23 @@ def read_source_as_df(  # type: ignore
         start_from: Statement | None = None,
         end_at: Statement | None = None,
     ) -> DataFrame:
-        pass
+        extra_options: dict = self.extra.copy().dict()
+        extra_options_with_prefix: dict = self._append_prefix(
+            prefix="kafka.",
+            subject=extra_options,
+        )
+        protocol_options: dict = self.protocol.get_options(self).copy()
+        extra_options_with_prefix.update(protocol_options)
+        options: dict = extra_options_with_prefix.copy()
+
+        if self.auth:
+            auth_options: dict = self.auth.get_options(self)
+            options.update(auth_options)
```

Comment:
```suggestion
        options = {
            f"kafka.{key}": value
            for key, value in self.extra.dict(by_alias=True, exclude_none=True).items()
        }
        options.update(self.protocol.get_options(self))

        if self.auth:
            options.update(self.auth.get_options(self))
```

Куча лишних копирований